src/zimcheck.cpp

```diff
@@ -437,23 +437,38 @@ void test_articles(const zim::File& f, ErrorLogger& reporter, ProgressBar progre
 
             auto links = getLinks(it->getData());
             std::unordered_map<std::string, std::vector<std::string>> filtered;
+            int nremptylinks = 0;
             for (const auto &l : links)
             {
                 if (l.front() == '#' || l.front() == '?') continue;
                 if (isInternalUrl(l) == false) continue;
+                if (l.empty())
+                {
+                    nremptylinks++;
+                    continue;
+                }
 
                 if (isOutofBounds(l, baseUrl))
                 {
                     std::ostringstream ss;
                     ss << l << " is out of bounds. Article: " << it->getLongUrl();
                     reporter.addError(TestType::URL_INTERNAL, ss.str());
+                    reporter.setTestResult(TestType::URL_INTERNAL, false);
```

Comment:
Is this really related to empty links detection ?

If not we should move it to a different commit.

Reply:
Done